README.md

```diff
@@ -2,14 +2,90 @@
 
 # Wikipedia KaiOS 
 
-The official Wikipedia KaiOS app.
+The official Wikipedia KaiOS app. [KaiOS](https://developer.kaiostech.com/) is a web-based mobile operating system that enables a new category of smart feature phones, a successor of the discontinued Firefox OS.
 
-Tasks and issues are tracked on [Phabricator](https://phabricator.wikimedia.org/project/profile/4305/).
+KaiOS applications are similar to Progressive Web Apps (PWAs):
+* Pure web applications
+* Manifest file
+* Installed on the device
+* Have access to the required APIs for notifications, offline, storage, audio, etc
+
+They are also different:
+* Format of the manifest is different
+* Size of the icons is specific to KaiOS
+* Need to be designed for a small, non-touch screen with a few physical buttons
+* Several APIs are specific to the platform (not web standards)
+* Running performance on the device
```

Comment:
These lines we could probably shave off if we want to keep the README lighter, for example

Reply:
I think we can remove the section of
1. `KaiOS applications are similar to Progressive Web Apps (PWAs)`
1. `They are also different`

---

> The official Wikipedia KaiOS app. [KaiOS](https://developer.kaiostech.com/) is a web-based mobile operating system that enables a new category of smart feature phones, a successor of the discontinued Firefox OS.

The first sentence information should be enough and there is a link for anyone who wants to learn more about KaiOS